cucumber/policy/features/support/policy/multifile2.yml

```diff
@@ -0,0 +1,35 @@
+- !user
```

Comment:
Overall I think we could probably reproduce the problem with smaller policies. That way they could be embedded directly in the Cucumber test code.

Reply:
This is probably true - I created these tests with the policies I found the error with, but though simple they are more complex than what is needed. I can trim them down if you like, but I'd prefer not to unless there's a real benefit to doing so.

Reply:
So, there is a real benefit I had in mind: There is a form of documentation generated from cucumber tests. You can see examples of it here : https://relishapp.com/conjur/possum-policy/docs/users-and-groups-can-be-added-to-groups

If the policies are embedded in the tests, then the generated cucumber documentation is a self-contained spec of the project's behavior. If the tests reference external files, then the generated documentation will not contain the policies, and so the user will have to guess, infer, or track down the Git repo of the project to really understand what's happening.

Reply:
I think this problem could be reproduced with policies that are just a few lines long. Given:

```
# groups.yml
- !group developers
```
```
# users.yml
- !user alice
- !grant
  role: !group developers
  members:
  - !user alice
```

If I load "groups" and "users" together (concatenated), and then "users" alone, then the exception you are testing for should be raised.